examples/twentyseven-pt-stencil-solver/twentyseven-pt-stencil-solver.cpp

```diff
@@ -334,6 +340,8 @@ void solve_system(const std::string &executor_string,
 
 int main(int argc, char *argv[])
 {
+    using ValueType = double;
```

Comment:
Up here, line `330`, missing `ResidualNormReduction<>` change. Same comment about the accuracy.